src/routes.js

```diff
@@ -29,4 +30,6 @@ app.post(
     addNewRequest
 );
 
+app.put('/request/:id', apiAuth, editRequest);
```

Comment:
Add in the middleware to validate the request body. You can use the same with one you used for the create request